>>> 'Pkg.add("Equations")' log
INFO: Cloning cache of Equations from https://github.com/jhlq/Equations.jl.git
INFO: Installing Equations v0.2.1
INFO: Package database updated
INFO: METADATA is out-of-date â€” you may not have the latest version of Equations
INFO: Use `Pkg.update()` to get the latest versions of your packages

>>> 'Pkg.test("Equations")' log
Julia Version 0.5.0-dev+4330
Commit 493157e (2016-05-26 09:11 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, haswell)
INFO: Testing Equations
WARNING: Method definition simplify(Equations.Expression) in module Equations at /home/vagrant/.julia/v0.5/Equations/src/tensors.jl:262 overwritten at /home/vagrant/.julia/v0.5/Equations/src/common.jl:597.
WARNING: Method definition print(Base.IO, Equations.Alt) in module Equations at /home/vagrant/.julia/v0.5/Equations/src/tensors.jl:336 overwritten at /home/vagrant/.julia/v0.5/Equations/src/printers.jl:61.
Error During Test
  Test threw an exception of type ErrorException
  Expression: randeval(sqrt(tri).rhs) == randeval(sqrt(:a ^ 2 + :b ^ 2))
  permutations(Union{Equations.Component,Equations.Expression,Number,Symbol}[:c,:c],) has been moved to the package Combinatorics.jl.
  Run Pkg.add("Combinatorics") to install Combinatorics on Julia v0.5-
   in permutations(::Array{Factor,1}, ::Vararg{Array{Factor,1},N}) at ./deprecated.jl:930
   in simplify(::Equations.Sqrt) at /home/vagrant/.julia/v0.5/Equations/src/sqrt.jl:26
   in sqrt(::Equations.Equation) at /home/vagrant/.julia/v0.5/Equations/src/sqrt.jl:6
   in include_from_node1(::String) at ./loading.jl:426 (repeats 2 times)
   in process_options(::Base.JLOptions) at ./client.jl:266
   in _start() at ./client.jl:322
ERROR: LoadError: LoadError: There was an error during testing
 in record(::Base.Test.FallbackTestSet, ::Base.Test.Error) at ./test.jl:321
 in do_test(::Base.Test.Threw, ::Expr) at ./test.jl:219
 in include_from_node1(::String) at ./loading.jl:426 (repeats 2 times)
 in process_options(::Base.JLOptions) at ./client.jl:266
 in _start() at ./client.jl:322
while loading /home/vagrant/.julia/v0.5/Equations/test/equationsTests.jl, in expression starting on line 16
while loading /home/vagrant/.julia/v0.5/Equations/test/runtests.jl, in expression starting on line 6
==============================[ ERROR: Equations ]==============================

failed process: Process(`/home/vagrant/julia/bin/julia -Cx86-64 -J/home/vagrant/julia/lib/julia/sys.so --compile=yes --check-bounds=yes --code-coverage=none --color=no /home/vagrant/.julia/v0.5/Equations/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: Equations had test errors
 in #test#49(::Bool, ::Function, ::Array{AbstractString,1}) at ./pkg/entry.jl:693
 in (::Base.Pkg.Entry.#kw##test)(::Array{Any,1}, ::Base.Pkg.Entry.#test, ::Array{AbstractString,1}) at ./null:0
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}})() at ./pkg/dir.jl:31
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}}, ::String) at ./file.jl:59
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./pkg/dir.jl:31
 in (::Base.Pkg.Dir.#kw##cd)(::Array{Any,1}, ::Base.Pkg.Dir.#cd, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./null:0
 in #test#3(::Bool, ::Function, ::String, ::Vararg{String,N}) at ./pkg.jl:245
 in test(::String, ::Vararg{String,N}) at ./pkg.jl:245
 in eval(::Module, ::Any) at ./boot.jl:225
 in process_options(::Base.JLOptions) at ./client.jl:243
 in _start() at ./client.jl:322

>>> End of log
